mobify/platform-scaffold
# Move PLP actions and parsing to the merlins connector

This PR moves the PLP actions and parsers into the merlin's connector

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1254
 **Linked PRs**: n/a

## Changes
- Added fetcher for PLP container
- Moved PLP parsers into the merlin's connector
- Moved product fetch action to the merlin's connector
- Deleted actions file for the products section of the store (no longer needed) 

## How to test-drive this PR
- Make sure you're using the merlins-connector in main.jsx
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- view any product list page and confirm the page still works as expected
